Runtime/Model/Database/BacktraceDatabaseRecord.cs

```diff
@@ -181,6 +181,10 @@ public virtual void Increment()
         /// <returns>Saved database record</returns>
         internal static BacktraceDatabaseRecord ReadFromFile(FileInfo file)
         {
+            if (!file.Exists)
```

Comment:
Null check?

Reply:
This is an internal method that our api call. We already know in higher level that file is defined.

```
   var files = BacktraceDatabaseFileContext.GetRecords();
            foreach (var file in files)
            {
                var record = BacktraceDatabaseRecord.ReadFromFile(file);
```